src/sw.ts

```diff
@@ -0,0 +1,43 @@
+async function askForAccessToken(client: Client): Promise<string> {
```

Comment:
return type of this function is `string`, but looking at the `postMessage` in `index.js` the `token` can be`null` so it should be changed with `Promise<string | null>`

Reply:
Actually more precisely `Promise<string | undefined>`